client/src/components/CatalogMainFilter/CatalogMainFilter.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { styled, Button, Box, Typography } from '@mui/material';
+import Grid from '@mui/material/Unstable_Grid2';
+import CloseIcon from '@mui/icons-material/Close';
+import { CatalogFilterPrice, CatalogFilterTourists, CatalogFilterRegion } from '../index';
+
+const FilterWrapper = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
```

Comment:
on a note:
You can get rid of these two properties just using `Stack` instead of `Box`
```suggestion
const FilterWrapper = styled(Stack)(({ theme }) => ({

```

Reply:
Done